catalog/confluence/catalog-confluence-source/pom.xml

```diff
@@ -116,13 +116,14 @@
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>ddf.security</groupId>
-            <artifactId>ddf-security-common</artifactId>
+            <groupId>ddf.lib</groupId>
+            <artifactId>gson-support</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>io.fastjson</groupId>
-            <artifactId>boon</artifactId>
```

Comment:
✏️ This pom property can be removed https://github.com/codice/ddf/blob/c12e493441c19ddb5e011fb2ff24896d6e1ead93/pom.xml#L170